Documentation/dl_build.md

```diff
@@ -64,4 +67,3 @@ If OK is printed, then etcd is working!
 [go]: https://golang.org/doc/install
 [build-script]: ../build
 [cmd-directory]: ../cmd
-
```

Comment:
Yeah but we can make it consistent rather than hardcoding the one in master branch:

```
[example hardware configuration]: ./op-guide/hardware.md#example-hardware-configurations
```